bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/RS232Connection.java

```diff
@@ -42,32 +38,36 @@
 
     private final Logger logger = LoggerFactory.getLogger(RS232Connection.class);
 
+    protected SerialPortManager serialPortManager;
     protected SerialPort serialPort;
 
     protected BufferedReader inputReader;
 
     protected RadioRAFeedbackListener listener;
     protected RS232MessageParser parser = new RS232MessageParser();
 
+    public RS232Connection(SerialPortManager serialPortManager) {
+        super();
+        this.serialPortManager = serialPortManager;
+    }
+
     @Override
     public void open(String portName, int baud) throws RadioRAConnectionException {
-        CommPortIdentifier commPort = null;
+        SerialPortIdentifier portIdentifier;
 
-        try {
-            commPort = CommPortIdentifier.getPortIdentifier(portName);
-        } catch (NoSuchPortException e) {
-            logAvailablePorts();
+        portIdentifier = serialPortManager.getIdentifier(portName);
```

Comment:
It's now possible to assign the variable where it is declared:

```suggestion
        SerialPortIdentifier portIdentifier = serialPortManager.getIdentifier(portName);
```